src/components/Search/styles.scss

```
@@ -0,0 +1,34 @@
+@import "../../styles/main.scss";
+
+.search {
+  float: right;
+  height: 100vh;
+  overflow: auto;
+  background-color: white;
+  @include flex-center();
```

Please add a newline here.
```suggestion
  background-color: white;
  
  @include flex-center();
```